test/suites/rbac.sh

```diff
@@ -0,0 +1,125 @@
+test_rbac() {
+  ensure_import_testimage
+
+  rbac_address="$(cat "${TEST_DIR}/rbac.addr")"
+  rbac_url="http://${rbac_address}"
+  key=$(curl -s "${rbac_url}/auth/discharge/info" | jq .PublicKey)
+  lxc config set rbac.api.key "${key}"
+  lxc config set rbac.api.url "${rbac_url}"
+  lxc config set "rbac.agent.url" "https://example.com"
+  lxc config set "rbac.agent.private_key" "${key}"
+  lxc config set "rbac.agent.public_key" "${key}"
```

Comment:
The private and public part are probably not meant to be identical.

Reply:
These don't really matter because the test server is not using TLS, they just need to be set for the driver to be loaded. I will replace them with nonsense and add a comment to explain.

Reply:
@monstermunchkin whilst looking into this I've found that the following config keys don't seem to be used at all for RBAC:
* `rbac.agent.username`
* `rbac.agent.url`
* `rbac.agent.private_key`
* `rbac.agent.public_key`

Should these be removed?

Reply:
This is my mistake. These are used for setting up authentication when the RBAC driver is loaded.